src/vl-rich-data-sorter.js

```diff
@@ -66,7 +54,14 @@ export class VlRichDataSorter extends VlElement(HTMLElement) {
     set direction(direction) {
         if (this.__direction !== direction) {
             this.__direction = direction;
-            this.__directionElement.setAttribute('icon', this._directionIcon);
+
+            const directionIcon = this._directionIcon;
+            if (directionIcon) {
+                this.__directionElement.setAttribute('icon', this._directionIcon);
+                this.__directionElement.classList.remove('vl-u-visually-hidden');
+            } else {
+                this.__directionElement.classList.add('vl-u-visually-hidden');
```

Comment:
ik dacht dat we de hele div niet zouden renderen in deze case ...

Reply:
Ok